README.md

```diff
@@ -595,7 +595,17 @@ The `bond` setting configures the options of bonded interfaces
 
 - `miimon`
 
-    Sets the MII link monitoring interval (in milliseconds)
+  Sets the MII link monitoring interval (in milliseconds)
+
+-`lacp_rate`
+
+  Specifies the rate at which link partners should transmit LACPDU packets in 802.3ad
+  mode. Possible values are `slow` or `fast`.
+
+-`xmit_hash_policy`
+
+  Selects the transmit hash policy used for slave selection in balance-xor and 802.3ad
+  modes. Possible values are `layer2`, `layer3+4` or `layer2+3`.
```

Comment:
This is the list of `xmit_hash_policy` in NetworkManager[1]:
```
static char const *const _option_default_strv_xmit_hash_policy[] =
    NM_MAKE_STRV("layer2", "layer3+4", "layer2+3", "encap2+3", "encap3+4", "vlan+srcmac");
```
[1] https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/blob/main/src/libnm-core-impl/nm-setting-bond.c#L150